releasing/lagom.md

```diff
@@ -82,25 +74,26 @@ See if there are [issues that need triage](https://github.com/issues?utf8=%E2%9C
   "Reviewing the Changes" above. This way you can make sure the GH milestone lists the correct information.
 * Ensure that the branch has a green build in [Travis CI](https://travis-ci.org/lagom/lagom/branches)
 * Create a pull request against the project with changes for the release:
-    * If it is a major or minor release, write a release blog post with highlights, contributions, etc. (Use the authors list you created above). This is not necessary for patch releases
+    * If it is a major or minor release, write [GitHub release notes](https://github.com/lagom/lagom/releases) with highlights, contributions, etc. (Use the authors list you created above). This is not necessary for patch releases
     * Update `currentLagomVersion`
     * If this is a MAJOR or MINOR version bump (RC or final), also update `currentDocsVersion`
     * If this is a MILESTONE or RC, update `previewVersions`
-* Copy the release announcement Markdown into [draft release notes on GitHub](https://github.com/lagom/lagom/releases).
 
 ## Publish the Artifacts
 
 ssh into `vegemite` and run:
 
     screen -r # try reconnecting to an existing, detached session; to force detach add -d, otherwise:
     screen sudo su - play # if you get disconnected, you can reconnect with screen -r
-    deploy/release --project lagom --branch <releaseBranch>
+    deploy/release --project lagom --branch <releaseBranch> --tag <new-tag>
+
+NOTE: `--tag <new-tag>` is meant for projects using `sbt-dynver` which also need a custom `release` script in their `sbt` build.
```

Comment:
What `release` script?  Does one such script exist in lagom/lagom?

Reply:
I forgot this process is specific to lagom. reverting.

Reply:
Reviewed wording. I think It's clearer now.